sale_weekday_delivery/models/res_partner.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+import calendar
+from datetime import timedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+WEEKDAY_MAPPING = {
```

Comment:
move this to a method and make strings translatable or add a second mapping which is translatable (in a method)

Reply:
As stated above, fixed by calling fields description when wwe have to return error messages